redaxo/src/core/lib/console/setup/run.php

```diff
@@ -0,0 +1,424 @@
+            if (!in_array($timezone, DateTimeZone::listIdentifiers(), true)) {
+                throw new InvalidArgumentException('Unknown timezone "'.$timezone.'" specified');
+            }
+            $config['timezone'] = $timezone;
+            $io->success('Timezone "'.$timezone.'" selected');
+        }
+
+        if (!$input->getOption('db-host') && !$input->getOption('db-login') && !$input->getOption('db-password') && !$input->getOption('db-name')) {
+            $io->section('Database information');
+            do {
+                $config['db'][1]['host'] = $io->ask('MySQL host', $config['db'][1]['host']);
+                $config['db'][1]['login'] = $io->ask('Login', $config['db'][1]['login']);
+                $config['db'][1]['password'] = $io->askHidden('Password');
+                $config['db'][1]['name'] = $io->ask('Database name', $config['db'][1]['name']);
+
+                $redaxo_db_create = $io->confirm('Create database', false);
+
+                rex::setProperty('db', $config['db']);
+                try {
+                    $err = rex_setup::checkDb($config, $redaxo_db_create);
+                } catch (PDOException $e) {
+                    $err = 'The following error occured: ' . $e->getMessage();
+                }
+
+                if ('' !== $err) {
+                    $io->error($err);
+                }
+            } while ('' !== $err);
+        } else {
+            $config['db'][1]['host'] = $input->getOption('db-host');
+            $config['db'][1]['login'] = $input->getOption('db-login');
+            $config['db'][1]['password'] = $input->getOption('db-password');
+            $config['db'][1]['name'] = $input->getOption('db-name');
+
+            $redaxo_db_create = true === $input->getOption('db-createdb');
+
+            rex::setProperty('db', $config['db']);
+            try {
+                $err = rex_setup::checkDb($config, $redaxo_db_create);
+            } catch (PDOException $e) {
+                $err = 'The following error occured: ' . $e->getMessage();
+            }
+
+            if ('' !== $err) {
+                $io->error($err);
+                return 1;
+            }
+        }
+        $io->success('Database connection successfully established');
+
+        // ---------------------------------- step 5 . create db / demo
+        $io->title('Step 5 of 6 / Database');
+
+        // Search for exports
+        $backups = [];
+        foreach (rex_backup::getBackupFiles('') as $file) {
+            if ('.sql' != substr($file, strlen($file) - 4)) {
+                continue;
+            }
+            $backups[] = substr($file, 0, -4);
+        }
+
+        $createdbOptions = [
+            'normal' => 'Setup database',
+            'override' => 'Setup database and overwrite it if it exitsts already (Caution - All existing data will be deleted!',
+            'existing' => 'Database already exists (Continue without database import)',
```

Comment:
Im Backend gibt es diese Option nur, wenn die Core-Tabellen in der DB gefunden wurden (deine `$tables_complete`-variable weiter unten).